Examples/Tests/spectral_staggered/inputs_hybrid_2d

```diff
@@ -5,7 +5,7 @@ amr.n_cell = 96 704
 warpx.numprocs = 1 2
 
 geometry.coord_sys = 0
-geometry.is_periodic = 0 0
+geometry.is_periodic = 1 0
```

Comment:
@EZoni here is the change in the input for the hybrid tests which now use `periodic`boundary for the transverse directions